app/konnect/reference/search.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Konnect Search
+content_type: reference
+description: Learn how to use the Konnect Search to search through all Konnect entities.
+---
+
+The {{site.konnect_short_name}} Search feature allows you to perform simple and advanced searches across all {{site.konnect_short_name}} entities within an organization. You can access search using the search bar (_command+k_) at the top of every page in {{site.konnect_short_name}} or using the [{{site.konnect_short_name}} Search API](/konnect/api/search/latest/).
+
+The {{site.konnect_short_name}} Search API is available in global and regional locations with regional-awareness, ensuring that returned entities are relevant to their geographical location. 
+
+Here are a few example use cases where you can use the {{site.konnect_short_name}} Search API in your automation: <!--I'm not sure, based on the examples, how you use the API for this and not just advanced search. these don't seem like automation use cases-->
+
+| You want to... | Then use... |
+| -------------- | ----------- |
+| Find entities that are "not compliant" | You can use search to find all routes that don't have a path that starts with `/api` |
+| Select from a list of available entities | For example, if you are attaching a route to a service, you can use search to find and identify which service to attach the route to. Or when you're adding members to a team, you can search for the user. |
+
+## Query Syntax
+
+The {{site.konnect_short_name}} Search API provides selectors, reserved characters, and logical operators that you can use to narrow your entity search. By combining these selectors, reserved characters, and logical operators, you can construct complex and precise queries to effectively use the {{site.konnect_short_name}} Search API.
+
+To perform a simple search, you can just search by the name of an entity, like a service, API product, or name of a team. You can also perform an advanced search using {{site.konnect_short_name}}'s query syntax to get more granular results.
+
+The following is an example advanced search query syntax:
+
+```
+type:team AND NOT label.department:eng AND name:*_qa
```

Comment:
@haydenlam I moved this query syntax section up and added a bit about an example query to explain how to formulate one. Does this work?